crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/service/CompareContributionService.java

```diff
@@ -50,6 +50,12 @@ && areContributionRecordsIdentical(contributionResult, activeContribution.get())
         return true;
     }
 
+    private boolean hasMagsCourtOutcomeChanged(MagCourtOutcome magsCourtOutcome, RepOrderDTO repOrderDTO) {
+        if (repOrderDTO == null) return false;
+
+        return !Objects.equals(magsCourtOutcome, MagCourtOutcome.getFrom(repOrderDTO.getMagsOutcome()));
```

Comment:
What is the equality you want to check here out of interest, that they're both the same object instance or that they're both equal using whatever the equality check is for a `MagCourtOutcome`?

Reply:
We need to check that the values for both mags court outcomes are equal, though both values can be passed through as null and this seemed to be the best option for performing a null safe check. If you know of a preferred way to do this though the happy to change it 👍

Reply:
Makes sense 👍